internal/commands/scan.go

```diff
@@ -597,30 +596,76 @@ func setupScanTypeProjectAndConfig(
 	info["project"].(map[string]interface{})["id"] = projectID
 	// Handle the scan configuration
 	var configArr []interface{}
-	if _, ok := info["config"]; !ok {
-		err = json.Unmarshal([]byte("[]"), &configArr)
-		if err != nil {
-			return err
+	resubmit, _ := cmd.Flags().GetBool("resubmit")
+	if resubmit {
+		logger.PrintIfVerbose("using latest scan configuration due to --resubmit flag")
```

Comment:
Same here with PrintfIfVerbose(logger.PrintIfVerbose("using latest scan configuration due to --%s flag", commonParams.ScanResubmit)

Reply:
Done, thanks !